src/kernel/src/main.rs

```diff
@@ -272,6 +242,7 @@ fn main() -> ExitCode {
     }
 }
 
+#[allow(non_snake_case)]
```

Comment:
Why do we need this?

Reply:
I don't want libSceLibcInternal to be lib_sce_libc_internal. It should probably be right above the variable.

Reply:
I think just `libc` should be okay to identify it because we only load one libc here.

Reply:
Done.